cni/network/network.go

```diff
@@ -518,18 +519,20 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 
 		ipamAddConfig := IPAMAddConfig{nwCfg: nwCfg, args: args, options: options}
 		if !nwCfg.MultiTenancy {
+			// what does this .Add() do?
```

Comment:
This is to retrieve ipamAddResult from ipamInvoker which is swiftv2 path; my PR is to move this part before